### ISSUE TITLE
Improve Hardware Wallet Integration

### ISSUE THREAD

**Issue by igorsilvacripto** [btcpayserver/btcpayserver-doc#1425]
I could not find the information I was looking for on the "Hardware Wallet Integration" page (`/HardwareWalletIntegration/`).
[PLEASE DESCRIBE HOW THE PAGE CAN BE IMPROVED]

Hello, I need some support with an issue that I haven't figured out yet. 

I have connected my BTCPay server with a hard wallet, I have a store, and I have also connected it to WooCommerce. 
What I have noticed though, is that: 

- the addresses generated to receive BTC do NOT match the address I see on my Ledger device. 
- I have replaced/removed wallet to try and do it again but it doesn't change. 
- The addresses don't even match the 9 addresses that are shown to confirm before I proceed. 
- The same thing happens on the website (WooCommerce).

I'd appreciate some advice, I have tried all I can to solve it but couldn't. Thanks!!!

**Comment by NicolasDorier** [comment]
@igorsilvacripto in the setup process, there is the last step which allow you to see the address on the device. If the device can show you the address, then it's fine.

The reason you don't see the same address may be (or may not be):
1. You are using different type of address between ledger and btcpay (legacy/segwit/wrapped segwit)
2. You already used a few addresses on your ledger, so ledger would show you the 25th address, but BTCPay, not knowing which address has been used, will show you the 1st one

The ultimate test is to try to send a few sats, then spend them from BTCPay with the BTCPay Vault.

**Comment by pavlenex** [comment]
@NicolasDorier also a rescan wallet may help or not with this?

**Comment by NicolasDorier** [comment]
For `2.`, yes

**Comment by igorsilvacripto** [comment]
Correct! I checked all addresses before confirming they were fine. Clicked on "Show on Device" and they were all displayed correctly on my Ledger. 

I'm selecting "segwit" on BTCPay, and my Ledger says it's a Native Segwit. Would that be correct? 

Last but not least... YES. I'm clicking on rescan wallet and it says I'm not the server administrator. Idk what that would be. My BTCPay Vault was open and running. 

Thank you guys for the response!

**Comment by NicolasDorier** [comment]
Ah you can't rescan the UTXO set because it's not your server then.
If it displayed on your Ledger then those address are correct and you are seeing is some sort gap limit issue (You can read more here https://www.youtube.com/watch?v=-7kHWZCYJkk)

That said, you are able to spend money received, and if the server crash, you can always reimport it on another server where you are admin, and rescan the utxo set.
